### PR TITLE
Html to text

### PR DESCRIPTION
**Pull request by timbrock**
Use .text() to add text rather than .html().
Also fixes a few small style issues.

### CODE REVIEW

File: src/update_graphic.js [hunk 1]
```diff
@@ -203,12 +203,12 @@ function clickHorse(d, i) {
...
-		}	
+		}
 	}
 	state.selected_horse = selected_horses.join();
-	console.log(selected_horses)
-	console.log(state.selected_horse)
-	
+	console.log(selected_horses);
+	console.log(state.selected_horse);
+
```
Comment (theduncanclark) [inline src/update_graphic.js]: Don't we want to just remove these?

Comment (timbrock) [reply src/update_graphic.js]: I would guess so.

Comment (theduncanclark) [reply src/update_graphic.js]: Kill, kill!

Comment (timbrock) [reply src/update_graphic.js]: Done.

**Pull request merged**